tests/test_port_add_remove.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+import time
+from dvslib.dvs_common import PollingConfig
+
+# the port to be removed and add
+PORT_A = "Ethernet64"
+PORT_B = "Ethernet68"
+
+"""
+DELETE_CREATE_ITERATIONS defines the number of iteration of delete and create to  ports,
+we add different timeouts between delete/create to catch potential race condition that can lead to system crush.
+"""
+DELETE_CREATE_ITERATIONS = 10
+
+@pytest.mark.usefixtures('dvs_port_manager')
+class TestPortAddRemove(object):
+
+    def test_remove_port_with_buffer_cfg(self, dvs, testlog):
```

Comment:
we should remove_port, add_port, then remove_port  with buffer_cfg to see if things are as expected.  This is to explicitly  excise the ref counters during those operations.  maybe we can have a test case like "test_remove_add_remove_port_with_buffer_cfg" ?

Reply:
@zhenggen-xu 
to me, this is a negative test that verifies a port will not be removed if there is a buffer configuration on it. We should keep this test.
So i would like to have the test `test_remove_add_remove_port_with_buffer_cfg` like this:
- when removing the port for the first time, remove port first and then the buffer configuration. verify the port will not be removed without buffer configuration cleared.
- when adding the port, add the port first and then the buffer configuration, as you suggested.
- when removing the port for the second time, remove the buffer configuration and then the port, as you suggested.
By doing so, we are able to verify both orders.
How do you think?

Reply:
@stephenxs  sounds good to me.

Reply:
Sure, looks good to me.